README.md

```diff
@@ -0,0 +1,732 @@
+| `if`   | `[{String : Any}]` | Yes      | Any additional caveats, such as regex matchers, contextual information (e.g. day of week), and so on |
+
+Here is an illustrative example:
+
+``` js
+{
+  // ...
+  "sub": "did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp"
+  "can": "blog/post/create",
+  "args": {
+    "tags": ["weekend", "news"],
+    "status": "draft",
+  },
+  "if": [
+    {
+      "day": "friday"
+    },
+    {
+      "field": "title",
+      "match": "/^From the newsroom:/"
+    },
+    {
+      "field": "body",
+      "max-chars": 2000
+    }
+  ]
+}
+```
+
+## 4.1 Subject
+
+The Subject MUST be the DID that initiated the delegation chain.
+
+For example:
+
+``` js
+{
+  "sub": "did:web:example.com",
+  // ...
+}
+```
+
+## 4.2 Resource
+
+Unlike Subjects and Commands, Resources are semantic rather than syntactic. The Resource is the "what" that a capability describes.
+
+By default, the Resource of a capability is the Subject. This makes the delegation chain self-certifying.
+
+``` js
+{
+  "sub": "did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp", // Subject & Resource
+  "can": "crud/update",
+  "args": {"status": "draft"}
+  // ...
+}
+```
+
+In the case where access to an [external resource] is delegated, the Subject MUST own the relationship to the Resource. The Resource SHOULD be referenced by a `uri` key in the relevant [Caveat(s)][Caveat], except where it would be clearer to do otherwise. This MUST be defined by the Subject and understood by the executor.
+
+``` js
+{
+  "sub": "did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp",
+  "can": "crud/create",
+  "args":{
+    "uri": "https://example.com/blog/", // Resource
+    "status": "draft"
+  },
+  // ...
+}
+```
+
+## 4.3 Command
+
+Commands MUST be presented as a case-insensitive string. By convention, Commands SHOULD be namespaced with a slash, such as `msg/send`.
+
+``` js
+{
+  "sub": "did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp": {
+  "can": "msg/send",
+  "arg": {
+    "from": "mailto:alice@example.com",
+    "to": "mailto:bob@example.com"
+  },
+  // ...
+}
+```
+
+Abilities MAY be organized in a hierarchy that abstract over many [Command]s. A typical example is a superuser capability ("anything") on a file system. Another is mutation vs read access, such that in an HTTP context, `write` implies `put`, `patch`, `delete`, and so on. [`*` gives full access][Wildcard Ability] to a Resource more in the vein of object capabilities. Organizing abilities this way allows Resources to evolve over time in a backward-compatible manner, avoiding the need to reissue UCANs with new resource semantics.
+
+### 4.3.1 Reserved Commands
+
+#### 4.3.1.1 `ucan/` Namespace
+
+The `ucan/` ability namespace MUST be reserved. This MUST include any ability string matching the regex `/^ucan\/.*/`
+
+Support for the `ucan/*` delegated proof ability is RECOMMENDED.
```

Comment:
This seems to be the only mention of the `ucan/*` ability. 
It should probably be explained, or be a link to an explanation somewhere.

Reply:
Hmm good point. Will expand.

Reply:
Done!